File: Server/WebSignalR/WebSignalR.Common/Entities/Room.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using WebSignalR.Common.Infrastructure;

namespace WebSignalR.Common.Entities
{
	public class Room : EntityBase
	{
		public Room()
		{
			ConnectedUsers = new SafeCollection<User>();
			ItemsToVote = new SafeCollection<VoteItem>();
		}

		private bool active;
		[Required]
		//[DataMember(IsRequired = true)]
		public bool Active
		{
			get { return active; }
			set { OnPropChanging("Active"); active = value; OnPropChanged("Active"); }
		}

		private string name;
		[Required]
		//[DataMember(IsRequired = true)]
		public string Name
		{
			get { return name; }
			set { OnPropChanging("Name"); name = value; OnPropChanged("Name"); }
		}

		private string descr;
		public string Description
		{
			get { return descr; }
			set { OnPropChanging("Description"); descr = value; OnPropChanged("Description"); }
		}

		public virtual ICollection<User> ConnectedUsers { get; set; }
		public virtual ICollection<VoteItem> ItemsToVote { get; set; }
	}
}

File: Client/Android/Agile.Coomon/src/com/udelphi/agile/common/Room.java
package com.udelphi.agile.common;

import java.util.ArrayList;
import java.util.List;

public class Room {

	public Room() {
		ConnectedUsers = new ArrayList<User>();
		ItemsToVote = new ArrayList<VoteItem>();
	}

	public int Id;
	public boolean Active;
	public String Name;
	public String Description;

	public List<User> ConnectedUsers;
	public List<VoteItem> ItemsToVote;

	public void AddUser(User usr) {
		ConnectedUsers.add(usr);
	}

	public void RemoveUser(User usr) {
		ConnectedUsers.remove(usr);
	}
	
	public void AddVote(VoteItem vote) {
		ItemsToVote.add(vote);
	}

	public void RemoveVote(VoteItem vote) {
		ItemsToVote.remove(vote);
	}
}
